src/Effects.js

```diff
@@ -1254,6 +1255,37 @@ module.exports = class Effects {
       }
     };
 
+    this.exploding_stars = {
+      stars: [],
+      initStars: function () {
```

Comment:
If we use `init` as the function name, I don't think we'll then need to call the `this.initStars()` on line 1277.